src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/PlacementRequestEntity.kt

```diff
@@ -90,6 +90,24 @@ interface PlacementRequestRepository : JpaRepository<PlacementRequestEntity, UUI
             WHERE
               bnm.placement_request_id = pq.id
           ) THEN 'unableToMatch'
+          WHEN EXISTS (
+            SELECT 
+                1 
+            FROM 
+                cas1_space_bookings sb 
+            WHERE
+                sb.placement_request_id = pq.id AND
+                sb.cancellation_occurred_at IS NULL
+          ) THEN 'matched'
+          WHEN EXISTS (
+            SELECT 
+                1 
+            FROM 
+                cas1_space_bookings sb 
+            WHERE
+                sb.placement_request_id = pq.id AND
+                sb.cancellation_occurred_at IS NOT NULL
+          ) THEN 'cancelled'          
```

Comment:
I don't think we need the 'cancelled' case. Looking at the API we take the status from there are only 3 potential options:

```
    @JsonProperty("notMatched") notMatched("notMatched"),
    @JsonProperty("unableToMatch") unableToMatch("unableToMatch"),
    @JsonProperty("matched") matched("matched")
```